src/js/components/Provenance/Catalogue/CatalogueCarouselDataset.tsx

```diff
@@ -0,0 +1,19 @@
+import { Card, Typography } from 'antd';
+import type { Dataset } from '@/types/metadata';
+import { useTranslationFn } from '@/hooks';
+
+const CatalogueCarouselDataset = ({ dataset }: { dataset: Dataset }) => {
+  const t = useTranslationFn();
+  return (
+    <Card style={{ height: '180px' }}>
```

Comment:
instead of being fixed height, this should expand to fill whatever container it is in